openmetadata-ui/src/main/resources/ui/src/components/BotListV1/BotListV1.component.tsx

```diff
@@ -200,7 +200,6 @@ const BotListV1 = ({
             <Switch
               checked={showDeleted}
               id="switch-deleted"
-              size="small"
```

Comment:
in your other PR, you are adding `size="small"` and here you are removing it!

Reply:
That size attribute is for tables this is for switch